dev-test/githunt/__generated__/comment.query.stencil-component.tsx

```diff
@@ -15,7 +15,13 @@ declare global {
     entry?: Types.Maybe<
       { __typename?: 'Entry' } & Pick<Types.Entry, 'id' | 'createdAt' | 'commentCount'> & {
           postedBy: { __typename?: 'User' } & Pick<Types.User, 'login' | 'html_url'>;
-          comments: Array<Types.Maybe<{ __typename?: 'Comment' } & CommentsPageCommentFragment>>;
+          comments: Array<
+            Types.Maybe<
```

Comment:
So we still have the fragment `CommentsPageCommentFragment` available as type, and it matches to the type here, right?

Reply:
yes! the fragment types are still generated, and the types match!